.github/workflows/ci.yml

```diff
@@ -120,7 +120,7 @@ jobs:
 
       # Tests
       - name: Pytest
-        run: pytest --cov --cov-report xml
+        run: pytest -n auto --dist loadscope --cov --cov-report xml
```

Comment:
why `loadscope`?

Reply:
Hi @fjetter, long time no see! :)

This is just being conservative. I'm not sure if there are any costly fixtures etc., or some test suites that expect that you run them all together. If it works out we may also drop that.

Reply:
Last time I worked with the code, it worked without. 

I don't have any strong objections. I was just asking because these kind of toggles have a tendency to stay around for a long time without anybody actually knowing what they are used for or if they are necessary